workflow/report/plot-heatmap.rst

```diff
@@ -1,3 +1,3 @@
 
-Heatmap of top 20 differentially expressed transcripts using the model ``{{ snakemake.params.model["full"] }}``, computed with sleuth.
+Heatmap of top 50 differentially expressed transcripts_genes using the model ``{{ snakemake.params.model["full"] }}``, from sleuth.
```

Comment:
I think apart from the number, no change is required in above text.

Reply:
Noted